carbon/carbon_informatie_4.pv

```diff
@@ -15,121 +15,49 @@ print years
 # data (van lage naar hoge waardes) 
 
 # agriculture
-data = gasData[0] 
-print data['sector']
+agri_data = gasData[0] 
```

Comment:
This and the following 8 lines:
used different variable names instead of just `data`, because the different data variables would be overridden and not used otherwise.